### PR TITLE
Privatize AP_Baro logging

### PR DESCRIPTION
**Pull request by hendjoshsr71**
Privatizes AP_Baro logging

SITL log output
private_baro

### CODE REVIEW

**Review by khancyr — APPROVED** [review]
Why not. Looks good

File: libraries/AP_Baro/AP_Baro.h [hunk 1]
```diff
@@ -197,6 +197,9 @@ class AP_Baro
     void set_log_baro_bit(uint32_t bit) { _log_baro_bit = bit; }
     bool should_log() const;
 
+    // Logging Functions
+    void Write_Baro(void);
+
```
Comment (peterbarker) [inline libraries/AP_Baro/AP_Baro.h:202]: Why isn't this private?

Comment (hendjoshsr71) [reply libraries/AP_Baro/AP_Baro.h]: Fixed. it should be.

**Comment by peterbarker** [comment]
Looks fine to me.

People might not want to do the separate-implementation-file thing for logging - worth it for EKFs, but perhaps not everywhere.  We'll see at the devcall.

**Comment by tridge** [comment]
looks good, just needs a rebase due to previous PR merge

**Review by tridge — APPROVED** [review]

**Comment by hendjoshsr71** [comment]
Rebased and ready to merge

**Comment by peterbarker** [comment]
Merged, thanks!

**Pull request merged**